src/averageAges.js

```diff
@@ -14,12 +14,21 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+
+const oneHundredYears = 100;
+const maleSex = 'm';
+const femaleSex = 'f';
```

Comment:
Static constants are usually named using uppercase.
```suggestion
const CENTURY = 100;
const MALE_SEX = 'm';
const FEMALE_SEX = 'f';
```